package.json

```
@@ -13,7 +13,7 @@
     "lint": "prettier --check \"src/**/*.ts\" \"test/**/*.ts\" \"docs/*.md\" *.md package.json tsconfig.json --end-of-line auto",
     "lint:fix": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\" \"docs/*.md\" *.md package.json tsconfig.json --end-of-line auto",
     "pretest": "tsc --noEmit -p test",
-    "test": "jest",
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules jest",
```

There is another way that doesn't use environment variables.


```suggestion
    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

---

Thanks, I implemented a slightly different version of this.